tests/test_ilastik/test_workflows/testCarvingGui.py

```diff
@@ -29,6 +29,10 @@
 import pytest
 
 
+if os.environ.get("ON_CIRCLE_CI", False):
```

Comment:
`None` is falsey too:

```suggestion
if os.environ.get("ON_CIRCLE_CI"):
```